src/libpgmoneta/http.c

```diff
@@ -29,99 +29,906 @@
 /* pgmoneta */
 #include <pgmoneta.h>
 #include <http.h>
+#include <logging.h>
+#include <network.h>
+#include <security.h>
 #include <utils.h>
 
-struct curl_slist*
-pgmoneta_http_add_header(struct curl_slist* chunk, char* header, char* value)
+/* system */
+#include <errno.h>
+#include <openssl/err.h>
+#include <unistd.h>
+
+int pgmoneta_http_add_header(struct http* http, char* name, char* value);
```

Comment:
I got to here...

Either they are in "public" API, and are called `pgmoneta_http_...` or they are static functions within the file called `http_...`